### PR TITLE
Create move API endpoint and handlers

### PR DESCRIPTION
**Pull request by amitch23**
## Description

This PR adds an API endpoint and handlers to create a move for a logged in user. A move will no longer be created for a user upon user creation (or, the first time a user logs in). The user is retrieved from the context rather than passing in a userID as a url param, and the selected move type is now required. 

## Verification Steps

* [x] All tests pass.
* [x] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI. NA
* [ ] This works in IE. NA

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/155838057) for this change

### CODE REVIEW

File: pkg/handlers/signed_certifications.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ func (h CreateSignedCertificationHandler) Handle(params certop.CreateSignedCerti
 		return response
 	}
 
+	// TODO: Get move from user
```
Comment (amitch23) [inline pkg/handlers/signed_certifications.go]: Remove this

File: swagger/internal.yaml [hunk 2]
```diff
@@ -149,6 +172,51 @@ paths:
...
 
 definitions:
+  CreateMovePayload:
+   type: object
+   properties:
+      selected_move_type:
+        type: string
+        enum:
+        - HHG
+        - PPM
```
[the remark this answers is not in the document]
Comment (amitch23) [reply swagger/internal.yaml]: done

File: pkg/handlers/signed_certifications.go (hunk 1 above)
Comment (amitch23) [reply pkg/handlers/signed_certifications.go]: done

File: pkg/handlers/moves_test.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	if len(moves) != 1 {
+		t.Errorf("Expected to find 1 move but found %v", len(moves))
+	}
+
+}
+
+func (suite *HandlerSuite) TestCreateMoveHandlerNoUserID() {
+	t := suite.T()
+	// Given: a user without authentication values in context
+	userUUID, _ := uuid.FromString("2400c3c5-019d-4031-9c27-8a553e022297")
```
[the remark this answers is not in the document]
Comment (amitch23) [reply pkg/handlers/moves_test.go]: done

File: swagger/internal.yaml [hunk 4]
```diff
@@ -149,6 +172,55 @@ paths:
...
+   properties:
+     id:
+       type: string
+       format: uuid
+       example: c56a4180-65aa-42ec-a945-5fd21dec0538
+     user_id:
+       type: string
+       format: uuid
+       example: c56a4180-65aa-42ec-a945-5fd21dec0538
+     selected_move_type:
```
Comment (stangah) [inline swagger/internal.yaml]: This and the identical field in `CreateMovePayload` should reference the same move type definition

Comment (amitch23) [reply swagger/internal.yaml]: done

**Pull request merged**